app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <p>If you're thinking about changing careers to become a teacher, knowing where to start and understanding what your options are can seem overwhelming. There's lots of dedicated support available to help you navigate the change.</p>
```

Comment:
```suggestion
    <p>If you're thinking about changing careers to become a teacher, there's lots of dedicated support available to help you navigate the change.</p>
```

Reply:
Above is just a suggestion, I like what you've written but thinking if we wanted to make it snappier this could be an option